di_website/whatwedo/blocks.py

```diff
@@ -81,3 +82,24 @@ class ExpertiseBlock(StructBlock):
 
     class Meta():
         template = 'blocks/expertise.html'
+
+
+class MapBlock(StructBlock):
+    """
+    Allows the addition of a single benefit
+    """
+    location = TextBlock()
+    address = TextBlock()
+    map = GeoBlock()
+
+    class Meta():
+        icon = 'fa-map'
+
+
+class MapStreamBlock(StreamBlock):
+    """
+    Handles creation of where we work locations
+    """
+    add_location = MapBlock()
```

Comment:
Just `location` is fine I think... can add any verbose naming for the UI in the meta class

Reply:
Also updated